src/types.rs

```diff
@@ -575,12 +578,14 @@ impl SourceMap {
     /// - `names`: a vector of names
     /// - `sources` a vector of source filenames
     /// - `sources_content` optional source contents
+    /// - `ignore_list` optional list of source indexes for devtools to ignore
     pub fn new(
         file: Option<Arc<str>>,
         mut tokens: Vec<RawToken>,
         names: Vec<Arc<str>>,
         sources: Vec<Arc<str>>,
         sources_content: Option<Vec<Option<Arc<str>>>>,
+        ignore_list: Option<Vec<u32>>,
```

Comment:
this is a breaking change

Reply:
That's a good point. This could be avoided by removing this parameter and using `add_to_ignore_list` in `SourceMapBuilder::into_sourcemap`.

Reply:
Could we add a corresponding `add_to_ignore_list` (or `set_ignore_list`) in the builder?